etc/WMAgentConfig.py

```diff
@@ -152,6 +152,7 @@
 config.DBS3Upload.dbsUrl = "OVERWRITE_BY_SECRETS"
 config.DBS3Upload.primaryDatasetType = "mc"
 config.DBS3Upload.dumpBlock = False  # to dump block meta-data into a json file
+config.DBS3Upload.gzipEncoding = False # swith to True when new DBSWriter will be in place
```

Comment:
typo. And how about documenting it like "set DbsApi requests to use gzip enconding, thus sending compressed data"?